modules/microshift-oc-mirror-list-ops-catalogs.adoc

```diff
@@ -6,19 +6,22 @@
 [id="microshift-oc-mirror-list-operators-catalogs_{context}"]
 = Inspecting catalog contents by using the oc-mirror plugin
 
-Use the following example procedure to select a catalog and list {OCP} Operators to add to your oc-mirror plugin image set configuration file. You must use oc mirror v1 to selecting a catalog and listing Operators.
+[role="_abstract"]
+Use the following example procedure to select a catalog and list {OCP} Operators to add to your oc-mirror plugin image set configuration file. You must use oc mirror v1 to select a catalog and list Operators.
```

Comment:
```suggestion
Use the following example procedure to select a catalog and list Operators to add to your oc-mirror plugin image set configuration file. You must use oc mirror v1 to select a catalog and list Operators.
```
They could be using a community catalog or their own custom catalog. Plus, the partner catalogs are RH-provided but made by ISVs. So, Operators is sufficient.